pymonetdb/mapi.py

```diff
@@ -120,7 +121,10 @@ def connect(self, database, username, password, language, hostname=None,
         self.database = database
         self.language = language
         self.unix_socket = unix_socket
-
+        if handshake_options:
+            self.handshake_options = [opt for opt in handshake_options]
```

Comment:
why do you loop over the list? can't you just assign? Or do you want to make a copy?

Reply:
Paranoia. it's not really necessary to make a copy of the list. I'll remove it.